charts/sscs-evidence-share/values.yaml

```diff
@@ -58,3 +58,4 @@ java:
     EMAIL_SMTP_TLS_ENABLED: "true"
     EMAIL_SMTP_SSL_TRUST: "*"
     UC_ENABLED: false
+    URGENT_HEARING_ENABLED: false
```

Comment:
do you want this here or in CPN flux so its just a config change to enable?

Reply:
by default it will be false, cnp flux change will override URGENT_HEARING_ENABLED flag